cmd/thanos/rule.go

```diff
@@ -775,6 +776,13 @@ func reloadRules(logger log.Logger,
 		}
 
 		files = append(files, fs...)
+		for _, fp := range fs {
+			if _, ok := seenFiles[fp]; ok {
```

Comment:
Is that necessary?

Reply:
Also don't see you describe this being a bug in PR desc :thinking: am I missing something?

Reply:
I was just picking up from the previous PR, as mentioned in the description, if you feel like it's not needed let me know! :)

Reply:
IIUC the idea is to remove duplicate rule files (e.g. `thanos rule --rule-file foo.yaml --rule-file foo.yaml`) but in practice it shouldn't matter as the rule manager will only load the file once eventually.

Reply:
That's true but I don't see a problem in avoiding processing the same file twice. I would say we should keep this change.